src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -36,16 +35,16 @@ public static bool IsKind<TNode>([NotNullWhen(returnValue: true)] this SyntaxNod
             return false;
         }
 
-        public static bool IsParentKind(this SyntaxNode node, SyntaxKind kind)
+        public static bool IsParentKind([NotNullWhen(returnValue: true)] this SyntaxNode? node, SyntaxKind kind)
```

Comment:
These annotations were needed as now they are called from nullable context, and callsites get incorrect nullable warnings without these annotations.